fluentd/run.sh

```diff
@@ -59,10 +59,8 @@ IPADDR6=`/usr/sbin/ip -6 addr show dev eth0 | grep inet6 | sed "s/[ \t]*inet6 \(
 export IPADDR4 IPADDR6
 
 BUFFER_SIZE_LIMIT=${BUFFER_SIZE_LIMIT:-1048576}
-MUX_CPU_LIMIT=${MUX_CPU_LIMIT:-500m}
-MUX_MEMORY_LIMIT=${MUX_MEMORY_LIMIT:-2Gi}
-FLUENTD_CPU_LIMIT=${FLUENTD_CPU_LIMIT:-100m}
 FLUENTD_MEMORY_LIMIT=${FLUENTD_MEMORY_LIMIT:-512Mi}
+MUX_MEMORY_LIMIT=${MUX_MEMORY_LIMIT:-2Gi}
```

Comment:
If we remove in-memory option entirely, if the disk is not available, should run.sh get failed and fluentd/mux fail to start?  Currently, memory buffer is used for fallback, as well...